test/jdk/javax/swing/SwingWorker/TestDoneBeforeDoInBackground.java

```diff
@@ -81,6 +83,10 @@ protected void done() {
             throw new RuntimeException("Cancel took too long: "
                                        + ((end - start) / 1000.0d) + " s");
         }
+        if (!doneLatch.await(CLEANUP_TIME + 1000L, TimeUnit.MILLISECONDS)) {
+            throw new RuntimeException("done didn't complete in time");
+        }
+        System.out.println("doInBackground " + doInBackground);
```

Comment:
```suggestion
        System.out.println("doInBackground = " + doInBackground);
```
Maybe add getState() too?